app/api/attendees.py

```diff
@@ -31,11 +35,26 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['ticket', 'event'], data)
 
-        ticket = db.session.query(Ticket).filter_by(id=int(data['ticket'])).first()
+        ticket = db.session.query(Ticket).filter_by(
+            id=int(data['ticket'])
+            ).first()
         if ticket is None:
-            raise UnprocessableEntity({'pointer': '/data/relationships/ticket'}, "Invalid Ticket")
+            raise UnprocessableEntity(
+                {'pointer': '/data/relationships/ticket'}, "Invalid Ticket"
```

Comment:
it's not that the ticket is invalid but more like the ticket was not found, isn't it?

Reply:
Yeah, I'll fix that. Thoughts @srv-twry @mayank8318 @iamareebjamal?

Reply:
yeah do it. :+1: